docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/1.configs-syntax.md

```diff
@@ -0,0 +1,95 @@
+# CONFIG syntax
+
+This document gives some introductions to `CONFIG` syntax in nGQL.
+
+## SHOW CONFIGS
+
+```ngql
+SHOW CONFIGS [ {META|GRAPH|STORAGE} ]
+```
+
+`SHOW CONFIGS` shows the configurations for the specified service. When the service is not specified, Nebula Graph returns all the service configurations. You do not need any privileges for the graph space to run this statement. But the returned results are different based on your account authentication. (TODO: privilege doc)
+
+For example:
+
+```ngql
+nebula> SHOW CONFIGS GRAPH;
++---------+---------------------------+-------+-----------+-------+
+| module  | name                      | type  | mode      | value |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "v"                       | "int" | "MUTABLE" | 0     |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "minloglevel"             | "int" | "MUTABLE" | 0     |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "slow_op_threshhold_ms"   | "int" | "MUTABLE" | 50    |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "heartbeat_interval_secs" | "int" | "MUTABLE" | 3     |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "meta_client_retry_times" | "int" | "MUTABLE" | 3     |
++---------+---------------------------+-------+-----------+-------+
+```
+
+```ngql
+nebula> SHOW CONFIGS META;
+Empty set (time spent 2059/3429 us)
+```
+
+The preceding query returns an empty set because the configurations for Meta Service are immutable in the Console.
+
+## GET CONFIGS
+
+```ngql
+GET CONFIGS [ {GRAPH|STORAGE} :] <configuration_name>
```

Comment:
configuration_name -> configuration_item?

Reply:
I think name is ok here